shared/chat/conversation/input-area/normal/platform-input.desktop.tsx

```diff
@@ -367,9 +355,9 @@ const styles = Styles.styleSheetCreate(
       emojiPickerContainer: Styles.platformStyles({
         common: {
           borderRadius: 4,
-          bottom: 34,
+          bottom: 32,
           position: 'absolute',
-          right: -22,
+          right: -64,
```

Comment:
This still has a different when the $ button is/is not present in the input box, but that bug is also present in master so not necessarily something that _has_ to be fixed in this pr

Reply:
I wonder why it didn't just use FloatingBox